### DISCUSSION TITLE
`useAtomsDevtools` not working anymore after migrating to jotai-devtools?

### DISCUSSION THREAD

**Discussion by nicobatalla** [pmndrs/jotai#1711] [Q&A] [answered]
We had been debugging just fine in v1.12.1 via redux devtools.

As soon as we moved to v1.13.1 and updated `useAtomsDevtools` to be from `jotai-devtools` actions are no longer tracked on the redux devtools.

Our implementation of the devtools is pretty simple, it looks like this:

`Component` code
```tsx
import { AtomsDevTools } from '@root/shared';
import { createStore } from 'jotai/vanilla';
import { Provider } from 'jotai/react';
import React, { useEffect } from 'react';

const store = createStore();

const MyComponent: React.FC = () => {
  useEffect(() => {
    // here we set initial values on the store using store.set()
  }, []);

  return (
    <Provider store={store}>
      <AtomsDevTools name="systemLogs">
        {/* page code */}
      </AtomsDevTools>
    </Provider>
  );
};

export default MyComponent;

```

`AtomsDevTools` wrapper component

```tsx
import React, { PropsWithChildren } from 'react';
import { useAtomsDevtools } from 'jotai-devtools';

export const AtomsDevTools: React.FC<PropsWithChildren<{name: string}>> = ({ children, name }) => {
  useAtomsDevtools(`jotai-debug-${name}`);
  return <>{children}</>;
};
```

This is the output on the redux devtools, nothing is tracked after `@@INIT`:

<img width="810" alt="image" src="https://user-images.githubusercontent.com/25337921/214453221-6e3e4e24-9a81-4f4b-9d11-9cad1f17e6a2.png">

**Comment by arjunvegda** [comment, marked as answer]
Thank you for the code snippets! 

What version of `jotai-devtools` are you using? We added support for Jotai's V2 API in [0.2.0-next.1](https://www.npmjs.com/package/jotai-devtools/v/0.2.0-next.1) version.

**Reply by nicobatalla** [reply]
Thanks for the quick answer!

We are using the latest from npm which is`0.1.0`

Just tested [0.2.0-next.1](https://www.npmjs.com/package/jotai-devtools/v/0.2.0-next.1) and it works perfect!

Do you plan to release that one as latest soon? Meanwhile, I can just point to 0.2.0-next.1

**Reply by arjunvegda** [reply]
Glad to hear 🎉! 

Yes, we're planning on doing a stable release for `jotai-devtools` after Jotai v2 is officially released

**Reply by nicobatalla** [reply]
Sounds good, thanks again!

**Reply by arjunvegda** [reply]
@nicobatalla - Circling back to let you know that `jotai@2.0.0` is released along with `jotai-devtools@0.2.0` 🎉 . Please let us know if you run into any issues
